security/certificate_types_descriptions/machine-config-operator-certificates.adoc

```diff
@@ -8,24 +8,43 @@ toc::[]
 
 == Purpose
 
-Machine Config Operator certificates are used to secure connections between the Red Hat Enterprise Linux CoreOS (RHCOS) nodes and the Machine Config Server.
+This certificate authority is used to secure connections from nodes to Machine Config Server during initial provisioning.
+
+There are two certificates: a self-signed CA ("the MCS CA") as well as a derived certificate ("the MCS cert").
+
+=== Provisioning details
+
+OpenShift installations using {op-system-first} are installed using Ignition.  This Ignition configuration is split into two parts; the first part is a "pointer" config which is just a reference to a URL for the full configuration served by the Machine Config Server (MCS).
+
+The "pointer" Ignition config manifests as e.g. the `worker.ign` file created by `openshift-install` for User Provisioned Infrastucture installs. For Installer Provisioned Infrastructure (IPI) installs using the Machine API Operator, this configuration appears as the `worker-user-data` secret.
 
 include::snippets/mcs-endpoint-limitation.adoc[]
 
 .Additional resources
 
 * xref:../../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[About the OpenShift SDN network plugin].
 
+=== Provisioning chain of trust
+
+The MCS CA is injected into the pointer configuration under the Ignition `security.tls.certificateAuthorities` configuration option. The Machine Config Server provides its "full" Ignition configuration via a webserver using the "MCS cert".
+
+Hence, this combination ensures that the client (Ignition running on a machine in the initramfs) is accessing the correct server because it validates that the MCS cert presented by the server has a chain of trust to an authority ("the MCS CA") it recognizes.
```

Comment:
```suggestion
The MCS cert presented by the server must have a chain of trust to an authority, the MCS CA, it recognizes. This ensures that the client, Ignition running on a machine in the initramfs, is accessing the correct server.
```

Reply:
Mmm...I think the new wording is wrong.  The MCS isn't recognizing the CA, the client is.
There may be more standard terms to use here.  Glancing at https://en.wikipedia.org/wiki/Transport_Layer_Security
we're talking about the phase here:

> The server usually then provides identification in the form of a [digital certificate](https://en.wikipedia.org/wiki/Public_key_certificate). The certificate contains the [server name](https://en.wikipedia.org/wiki/Hostname), the trusted [certificate authority](https://en.wikipedia.org/wiki/Certificate_authority) (CA) that vouches for the authenticity of the certificate, and the server's public encryption key.

The key (pun intended) thing here is that the client is verifying the server.

Reply:
gotcha, I was iffy on this too. appreciate the clarification

Reply:
```suggestion
The client, Ignition running on a machine in the initramfs, validates that the MCS cert presented by the server has a chain of trust to an authority, the MCS CA, it recognizes. This ensures that the client is accessing the correct server.
```

Reply:
LGTM